Makefile

```diff
@@ -156,6 +156,9 @@ serve-docs: docs ## serves the project documentation from an HTTP server
 .PHONY: serve-docs
 
 ifeq ($(IN_DOCKER),true)
+ifeq (, $(shell which docker))
+	$(error "Docker does not seem installed, please install docker first.")
+endif
```

Comment:
This doesn't work for me on my MacOS workstation.

I think you might need to do something like:

```
...
HAS_DOCKER := $(shell command -v docker 2> /dev/null)
...

image:
    ifndef HAS_DOCKER
        $(error "Docker does not seem to be installed....")
    endif
    # other stuff

```

Reply:
What is your error exactly?

Reply:
It was the same error as reported in the original issue (#1397)